### PR TITLE
Update README, Site-wide Settings and User Guide

### LINKED ISSUE #42: Update README Details
- Update all contents to match your own project.

- Update the link of the GitHub Actions build status badge (Build Status) so that it reflects the build status of your team repo.

- Acknowledge the original source of the code e.g., `This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org).`

### LINKED ISSUE #43: Update Project Site-wide Settings

### LINKED ISSUE #51: Update User Guide: Adding a student, Editing a student, Delete

### PR DESCRIPTION
**Pull request by chaiwanlin**
* Adapted README to EdRecord project. (Closes #42)
* Updated site-wide settings to EdRecord project and repository. (Closes #43)
* Updated Adding a student, Editing a student, Delete a student features in User Guide. (Closes #51)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,27 @@
...
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+EdRecord is a **sample project** for **teaching staff (e.g. TAs, profs etc.) who prefer CLI to GUI and have to
```
Comment (marcusc55) [inline README.md:5]: I think we can change the "sample project" to something like "CS2103 project".

**Review by marcusc55 — CHANGES_REQUESTED** [review]
Just a small change, otherwise LGTM!

File: README.md (hunk 1 above)
Comment (chaiwanlin) [reply README.md]: Aights done!

**Review by caipng — CHANGES_REQUESTED** [review]
Small change, other than that LGTM

**Review by marcusc55 — APPROVED** [review]
LGTM!

**Review by caipng — APPROVED** [review]
LGTM!

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (chaiwanlin) [reply README.md]: Updated!

**Pull request merged**